resources/function_help/json/ltrim

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "ltrim",
+  "type": "function",
+  "groups": ["String"],
+  "description": "Removes the longest string containing only characters in characters (a space by default) from the start of string.",
```

Comment:
```suggestion
  "description": "Removes the longest string containing only the specified characters (a space by default) from the start of string.",
```
`characters in characters` looks weird to me. But I'm also worrying about how this could be translated, considering that translators ignore the actual context (and may translate the second one)

Reply:
Good point!